src/snowflake/snowpark/_internal/analyzer/analyzer.py

```diff
@@ -449,7 +451,7 @@ def analyze(
                     expr.child, df_aliased_col_name_to_real_col_name, parse_local_name
                 ),
                 expr.direction.sql,
-                expr.null_ordering.sql,
+                expr.null_ordering.sql,  # type: ignore [attr-defined]
```

Comment:
mypy thinks the `expr. null_ordering` is an `object` instead of an `NullOrdering` instance.